pkg/formatter/table.go

```diff
@@ -30,13 +30,30 @@ import (
 	"github.com/chef/chef-analyze/pkg/reporting"
 )
 
-const MinTermWidth = 120
+const (
+	MinTermWidth                   = 120
+	filteredEmptyCookbookResultFmt = "\nNo cookbooks were found in the run lists of any filtered nodes.\n" +
```

Comment:
nit: move all these messages to the `errors.go` constant file?

Reply:
Will make this a separate Pr, and will do it for all messages.